examples/hierarchical-env/settings.rs

```diff
@@ -1,4 +1,5 @@
 use config::{Config, ConfigError, Environment, File};
+use serde_derive::Deserialize;
```

Comment:
Why is this import necessary if it wasn't before?

Reply:
Never mind... because you removed the `#[macro_use]` import in the `main.rs` file!